build.tmpl.yml

```diff
@@ -228,66 +225,134 @@ archives:
   - id: windows-cgo
     builds:
       - windows-cgo
-    replacements:
-      darwin: macOS
-      386: 32bit
-      amd64: 64bit
     format_overrides:
       - goos: windows
         format: zip
-    name_template: "{{ .ProjectName }}_{{ .Version }}-{{ .Os }}_sqlite_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
+    name_template: >-
+      {{- .ProjectName }}_
+      {{- .Version }}-
+      {{- if .Os eq "darwin" }}macOS
+      {{- else }}{{ .Os }}
+      {{- end }}-sqlite_
+      {{- if .Arch eq "amd64" }}64bit
+      {{- else if .Arch eq "386" }}32bit
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end }}
 
   - id: macos-cgo
     builds:
       - macos-cgo
-    replacements:
-      darwin: macOS
-      386: 32bit
-      amd64: 64bit
-    format_overrides:
-      - goos: windows
-        format: zip
-    name_template: "{{ .ProjectName }}_{{ .Version }}-{{ .Os }}_sqlite_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}"
-
-  - id: linux-cgo
+    name_template: >-
+      {{- .ProjectName }}_
+      {{- .Version }}-
+      {{- if .Os eq "darwin" }}macOS
+      {{- else }}{{ .Os }}
+      {{- end }}-sqlite_
+      {{- if .Arch eq "amd64" }}64bit
+      {{- else if .Arch eq "386" }}32bit
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end }}
+
+  - id: linux-cgo-amd64
```

Comment:
Is there a reason why we split these two? If not, please revert the split and keep all the IDs the same.